packages/graphql-hooks-memcache/index.d.ts

```diff
@@ -0,0 +1,16 @@
+export = MemCacheFunction
+
+declare function MemCacheFunction(options?: {
+  size?: number
+  ttl?: number
+  initialState?: object
+}): MemCache
+
+interface MemCache {
+  get(keyObject: object): object
```

Comment:
Technically `keyObject` could be `any`, but `graphql-hooks` gives it an object 😆